### ISSUE TITLE
BoundPeer.ToString() shows too many things and takes long lines

### ISSUE THREAD

**Issue by longfin** [planetarium/libplanet#834]
Currently, `BoundPeer.ToString()` seems like in logs as below

```
[10:13:48 DBG] Removing peer 0x9aed323473f04229a20C3DbFaacb80C75AC0dDaE.Unspecified/15.165.103.105:64391.1 (Bencodex.Types.Dictionary {
  "downloadUrls": {
    "macOS": "",
    "Windows": ""
  },
  "timestamp": "2020-03-25T07:39:53.8075281+00:00"
}) from table.
```

It adds new lines to messages and complicates them. so it would be better if we simplify it.

**Comment by limebell** [comment]
It is OK with removing only new lines? Or should hide some information that `Peer.ToString()` contains?

**Comment by longfin** [comment]
> It is OK with removing only new lines? Or should hide some information that `Peer.ToString()` contains?

I guess it's still too verbose if only removing newlines. so it would be better we can abstract some info in `Peer.ToString()`.

**Comment by dahlia** [comment]
IMHO fields like `Extra` would be better to be hidden.

**Comment by dahlia** [comment]
Seems fixed in 891c3fbbd3719047f3d5ce0fbc062511a749b025.
